script.js

```diff
@@ -0,0 +1,33 @@
+function calculation() {
+  let operator = document.getElementById('operator')
+  let mathOperators = operator.options[operator.selectedIndex].value
```

Comment:
this line can be replaced by below i think:

`let mathOperators = operator.value`